.vscode/extensions.json

```
@@ -0,0 +1,15 @@
+{
+  // List of extensions which should be recommended for users of this workspace.
+  "recommendations": [
+    "golang.go",
+    "editorconfig.editorconfig",
```

```suggestion
    "EditorConfig.EditorConfig",
```
see https://github.com/open-vsx/publish-extensions/pull/526#issuecomment-997284580